lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -69,86 +71,38 @@ function make_prob_vector(H::Type{<:AbstractRydberg}, V::UpperTriangular{T}, Ω:
     return ops, p, energy_shift
 end
 
+total_hx(H::Rydberg)::Float64 = sum(H.Ω) / 2
+haslongitudinalfield(H::AbstractRydberg) = !iszero(H.δ)
 
-###############################################################################
-
-# function BlockadeRydberg(dims::NTuple{N, Int}, J::Float64, hx::Float64, hz::Float64, pbc=true) where N
-# end
+function _make_vector(param::Real, Ns::Int64)
```

Comment:
```suggestion
function _make_vector(param::Real, Ns::Int)
```
Use Default integer type `Int`